### PR TITLE
[EN] basics 2.3.0

### PR DESCRIPTION
**Pull request by starfish719**
Example -> Friends

Maybe, this pullrequest fix wasn't enough

https://github.com/elixirschool/elixirschool/commit/6eee4a0b293bd5ef98d026efc8cc2bb9b5bbb975#diff-a3d9528d5c48b91d7a98fe56337f1bb8

### CODE REVIEW

**Comment by starfish719** [comment]
ping @elixirschool/english

File: en/lessons/ecto/basics.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-version: 2.2.0
+version: 2.2.1
```
Comment (doomspork) [inline en/lessons/ecto/basics.md]: It looks like we need to apply this change to a number of translation so maybe this should justify `2.3.0`?

Comment (starfish719) [reply en/lessons/ecto/basics.md]: @doomspork Thank you for your review :)
I changed version 👍 
https://github.com/elixirschool/elixirschool/pull/1923/commits/193ba4e18acc77e90d2b08238e47228a16b7cfdb

**Comment by doomspork** [comment]
Thank you for all your help @starfish719, you're awesome! 💜

**Pull request merged**